settings.gradle

```diff
@@ -52,6 +52,12 @@ project(':nessie').name = 'iceberg-nessie'
 project(':gcp').name = 'iceberg-gcp'
 project(':dell').name = 'iceberg-dell'
 
+if (null != System.getProperty("allVersions")) {
+  System.setProperty("flinkVersions", System.getProperty("knownFlinkVersions"))
+  System.setProperty("sparkVersions", System.getProperty("knownSparkVersions"))
+  System.setProperty("hiveVersions", System.getProperty("knownHiveVersions"))
```

Comment:
is it intentional to now have scalaVersions here?

if not, we can add it -- https://github.com/apache/iceberg/pull/10478